### ISSUE TITLE
Text in address bar is lost

### ISSUE THREAD

**Issue by iondissonance** [QupZilla/qupzilla#2365]
How to reproduce:

1. Go to this link https://www.google.ru/search?client=qupzilla&q=writeback+throttling&gws_rd=cr&ei=k8VGWY-vCOLO6ATvspnoBA#newwindow=1&q=blk+wbt++opensuse

2. Click on "Packages changed: kernel-source (4.4.70 -> 4.4.69) === Details ..." link  or any other(!)

3. Look at address bar

**Comment by iondissonance** [comment]
https://youtu.be/Hf6kW6GlDfw video

**Comment by iondissonance** [comment]
Any other link from google/yandex searching results and may be more, more sites

**Comment by cranes-bill** [comment]
Hm, something very odd here:
I can't reproduce that on my system, but can reproduce it on the same system running on another partition. Perhaps related to some desktop settings...!?

**d6b142bba2 - qt-5.9**

**Comment by cranes-bill** [comment]
Oooops, i was wrong, sorry! The situation is follow:
I can reproduce with direct click, but can't reproduce with middle click.

**Comment by nekoswag** [comment]
Probably related - since Qt 5.9 on Tumbleweed, the address bar doesn't work as a search bar anymore.

**Comment by KottV** [comment]
And even more. Clicking an item from pulled down address bar doesn't work. Selecting with keys - works.
Tumbleweed Qt5.9

**Comment by nowrep** [comment]
> Probably related - since Qt 5.9 on Tumbleweed, the address bar doesn't work as a search bar anymore.

@nekoswag You mean typing some text into address bar and then pressing enter doesn't search for the typed text? Works fine for me with Qt 5.9.1

**Comment by nowrep** [comment]
Can I get a better testcase for the original bug? I can't reproduce the issue neither with original link nor duckduckgo.com with links opened in new tab.

**Comment by JuPlutonic** [comment]
I run couple of month Qt 5.9.0, d6b142b  - 2.1.99 such problems. This TXT file opens with address in address bar from yandex.ru/google.ru

**Comment by KottV** [comment]
fixed on Tumbleweed, confirm
thanx

**Comment by Vistaus** [comment]
No problem here on Solus either with Qt 5.9.1 and QupZilla 2.1.99

**Comment by eht16** [comment]
FWIW, using Qupzilla 2.1.2 and QtWebEngine 5.9.1, I have the same problem as above: the address bar search is broken. I've configured Searx as default engine on a custom URL.
Whenever I press Enter on the address bar with some search text, I see the following message on stderr:

`js: Form submission canceled because the form is not connected`

I also tried to configure the default search engine as POST and GET, assuming the error message is related to POST requests but it didn't make any difference.
